lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -104,6 +107,8 @@ export class TagsCreatorComponent implements OnInit, OnDestroy {
         return this._tags;
     }
 
+    tagsToDisplay: Chip[] = [];
```

Comment:
This field should be defined below inputs and outputs. You defined it between inputs and outputs.